recipes/userspace-rcu/all/conanfile.py

```diff
@@ -1,8 +1,8 @@
 import os
 
 from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conan.tools.files import get, rmdir
 from conans.errors import ConanInvalidConfiguration
-from conan.tools.gnu.pkgconfigdeps.pc_files_creator import get_pc_files_and_content
 
 required_conan_version = ">=1.33.0"
```

Comment:
For `rmdir` https://github.com/conan-io/conan/releases/tag/1.47.0

```suggestion
required_conan_version = ">=1.47.0"
```